eth_tester/backends/pyevm/utils.py

```diff
@@ -17,10 +17,6 @@ def get_pyevm_version():
         return None
 
 
-def is_pyevm_available():
-    pyevm_version = get_pyevm_version()
-
-    if pyevm_version is None:
-        return False
-    else:
-        return True
+def is_supported_pyevm_version_available():
+    version = get_pyevm_version()
+    return version and version >= Version("0.5.0")
```

Comment:
This is the version that would be nice to dynamically get.
I can think of a way to do it but that involves creating a source-of-truth that both here and `setup.py` can use... However, it seems like there should be a way using `pkg_resources` or something but I could not figure out a way.

Reply:
Yeah, I don't see a nice way to do it either. I think this is good enough for now 👍